public/i18n/en/translation.json

```diff
@@ -952,7 +956,16 @@
       "confirm_delete_rule": "You are about to delete DPI rule on interface '{iface}'",
       "delete_rule": "Delete rule",
       "blocked_apps_and_protocols": "Blocked apps and protocols",
-      "plus_num_others": "+{num} others"
+      "plus_num_others": "+{num} others",
+      "delete_exception_message": "You are about to delete DPI exception '{name}'",
+      "delete_exception": "Delete exception",
+      "add_exception": "Add exception",
+      "edit_exception": "Edit exception",
+      "exceptions_description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras bibendum aliquam tortor nec ultricies.",
```

Comment:
@gsanchietti any suggestion on what to put here?
![image](https://github.com/NethServer/nethsecurity-ui/assets/8052641/4bad9766-14ff-4b18-91c9-27277abc392c)